alerts/machine-config-operator/MachineConfigDaemonPivotError.md

```diff
@@ -0,0 +1,151 @@
+# MCDPivotError
+
+## Meaning
+
+Alerts the User when the Machine Config Daemon
+is either unable to or detecs an error when
+trying to pivot OS image to a new version.
```

Comment:
Many of the other runbooks start with _This alert is triggered_ or similar wording
```suggestion
This alert is triggered when the Machine Config Daemon
detects an error when trying to pivot OS image to a new version
or is unable to pivot to a new version.
```

Reply:
I guess the customer could also manually pivot it (Not sure what the consequences of doing that outside of trying to finish an upgrade would be though...) I think the key thing to note is if the os image is updated it will do a pivot. An upgrade is just the most relevant and common example that most customers will run into.